### ISSUE TITLE
a False Positives example

### ISSUE THREAD

**Issue by Abirdcfly** [timonwong/logrlint#7]
```bash
$ go run -v main.go
Found existing alias for "go run". You should use: "gor"
command-line-arguments
"level"=0 "msg"="default info log" "stringVal"="value" "intVal"=12345
"level"=0 "msg"="default info log" "key1"="<no-value>"
"level"=0 "msg"="default info log" "key1"="val1"
$ logrlint ./...
/Users/fupeng/go/src/tmpa/main.go:26:29: odd number of arguments passed as key-value pairs for logging
/Users/fupeng/go/src/tmpa/main.go:28:29: odd number of arguments passed as key-value pairs for logging
```

main.go
```go
// https://github.com/go-logr/logr/blob/master/example_test.go
package main

import (
	"fmt"

	"github.com/go-logr/logr"
	"github.com/go-logr/logr/funcr"
)

// NewStdoutLogger returns a logr.Logger that prints to stdout.
func NewStdoutLogger() logr.Logger {
	return funcr.New(func(prefix, args string) {
		if prefix != "" {
			_ = fmt.Sprintf("%s: %s\n", prefix, args)
		} else {
			fmt.Println(args)
		}
	}, funcr.Options{})
}

func main() {
	l := NewStdoutLogger()
	l.Info("default info log", "stringVal", "value", "intVal", 12345)
	t := make([]interface{}, 0)
	t = append(t, "key1")
	l.Info("default info log", t...)  // should check
	t = append(t, "val1")
	l.Info("default info log", t...)  // False Positives here ! 
	// Output:
	// "level"=0 "msg"="default info log" "stringVal"="value" "intVal"=12345
	// "level"=0 "msg"="V(0) info log" "stringVal"="value" "intVal"=12345
	// "msg"="error log" "error"="an error" "stringVal"="value" "intVal"=12345
}

```

**Comment by timonwong** [comment]
Thanks for reporting, for now, I think it's better to prevent checking the ellipsis args

**Comment by timonwong** [comment]
ellipsis is now skipped. for golangci-lint users, I'll fire a PR later
